src/index.ts

```diff
@@ -25,20 +25,7 @@ export function useApp(
   }
 }
 
-export function useBootstrap(store: Store<State>) {
-  return async (isAllowedToMakeApiCalls: boolean = true) => {
-    if (isAllowedToMakeApiCalls) {
-      await Promise.all([
-        // Fetches basic resources before setting up the router and mounting the
-        // application. This is mainly needed to properly redirect users to the
-        // onboarding flow in the appropriate scenarios.
-        store.dispatch('bootstrap'),
-        // Loads available policy types in order to populate the necessary
-        // information used for and policy lookups in the app.
-        store.dispatch('fetchPolicyTypes'),
-      ])
-    } else {
-      store.state.defaultVisibility.appError = false
-    }
+export function useBootstrap() {
```

Comment:
Why is this still around btw?

Reply:
Yeah not super happy about it either, but didn't want to rearrange too much here.

> Why is this still around btw?

In production its a noop, but in development this empty function gets decorated with MSW functionality. Longer term we will probably move MSW elsewhere and then we can get rid of this entirely.

Reply:
Could you make this explanation a comment in the code so that someone doesn't remove what seems to be deadcode?

Reply:
Oh yeah sure makes sense, sorry just saw this! 2 tics